### DISCUSSION TITLE
[Release Vote] Vote for Release V0.4.1-rc2

### DISCUSSION THREAD

**Discussion by baiyangtx** [apache/amoro#1290] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Arctic 0.4.1 release.

https://github.com/NetEase/arctic/commit/64425ecb44fdb7c420688c837b84e503cdbf1c60

This corresponds to the tag: v0.4.1-rc2
https://github.com/NetEase/arctic/commits/v0.4.1-rc2
https://github.com/NetEase/arctic/tree/v0.4.1-rc2
Please check out, build, and test.

[ ] +1 Release this as Arctic 0.4.1
[ ] -1 Do not release this because...

**Comment by baiyangtx** [comment]
I tested in local and run the quick start demo. all is ok

**Comment by zhoujinsong** [comment]
+1

For testing the release in my local environment with the following step:

- Ran build and test with JDK8 and JDK11
- Ran the quick start guild
- Created the iceberg format catalog and table

**Comment by YesOrNo828** [comment]
+0 
What I did:
- Ran `mvn -DskipTests package -P toolchain` succeed;
- Ran Quick Demo succeed;
- Check that the last commit id is correct;
- Creating this issue could be improved in the feature. #1303

**Comment by shidayang** [comment]
+1 
I run all Trino query im 0.4.1, All is ok

**Comment by wangtaohz** [comment]
+1

I tested it locally :

- Ran optimizing integration test locally
- Installed the local ams environment, and ran the quick start guild
- Tested Optimizing of Iceberg format table and Mixed Iceberg format table

For 0.4.1, the Self-Optimizing may have a larger time interval than what is shown in doc.

<img width="1704" alt="image" src="https://user-images.githubusercontent.com/103108928/228758072-43414a8a-8bdd-4963-be23-45c4cf60d6fc.png">

**Comment by Aireed** [comment]
+1

I tested it locally :

- compile locally
- ran flink-runtime according to sql-client
- run quick start locally
- tested local optimizer and mixed iceberg table

For 0.4.1 , our site doc is out of date, we should update it.
